src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sylius.active_promotions_by_channel_provider" class="Sylius\Component\Core\Provider\ActivePromotionsByChannelProvider">
```

Comment:
We should start to clean service name conventions. Even in this file we have two different ones. Can we be change some of them to keep some consistency? 

`sylius.promotion.active_promotions_by_channel_provider`?